docs/guide/blog/comment-blog.md

```diff
@@ -0,0 +1,607 @@
+
+confirm transaction before signing and broadcasting [y/N]: y
+```
+
+The transaction output:
+
+```bash
+{"height":"6861","txhash":"6086372860704F5F88F4D0A3CF23523CF6DAD2F637E4068B92582E3BB13800DA","codespace":"","code":0,"data":"0A100A0A437265617465506F737412020801","raw_log":"[{\"events\":[{\"type\":\"message\",\"attributes\":[{\"key\":\"action\",\"value\":\"CreatePost\"}]}]}]","logs":[{"msg_index":0,"log":"","events":[{"type":"message","attributes":[{"key":"action","value":"CreatePost"}]}]}],"info":"","gas_wanted":"200000","gas_used":"44674","tx":null,"timestamp":""}
+```
+
+Create a comment:
+
+```bash
+blogd tx blog create-comment 0  Uno "This is the first comment" --from alice
+```
+
+```bash
+{"body":{"messages":[{"@type":"/cosmonaut.blog.blog.MsgCreateComment","creator":"cosmos17pvwgu36fu37j8y9gc4pasxsj3p26ghmlqvngd","id":"0","title":"Uno","body":"This is the first comment","blogID":"2","createdAt":"0"}],"memo":"","timeout_height":"0","extension_options":[],"non_critical_extension_options":[]},"auth_info":{"signer_infos":[],"fee":{"amount":[],"gas_limit":"200000","payer":"","granter":""}},"signatures":[]}
+```
+
+When prompted, press Enter to confirm the transaction:
+
+```bash
+confirm transaction before signing and broadcasting [y/N]: y
+```
+
+The command output shows the results of the transaction:
+
+```bash
+code: 0
+codespace: ""
+data: 0A270A252F636F736D6F6E6175742E626C6F672E626C6F672E4D7367437265617465436F6D6D656E74
+gas_used: "45891"
+gas_wanted: "200000"
+height: "118"
+info: ""
+logs:
+- events:
+  - attributes:
+    - key: action
+      value: CreateComment
+    type: message
+  log: ""
+  msg_index: 0
+raw_log: '[{"events":[{"type":"message","attributes":[{"key":"action","value":"CreateComment"}]}]}]'
+timestamp: ""
+tx: null
+txhash: 0CAFC113D1C73BC0210EFEA5964EBD2EB530311169FB442C5CBF0B5E92521C41
+```
+
+## Display post and comment
+
+```bash
+blogd q blog comments 0
+```
+
+The results are output:
+
+```bash
+Comment:
+- body: Let us add random comment
+  createdAt: "14094"
+  creator: cosmos1g7x9cpj6w0jklshe3se57tlwydx6yfl8ex5g7n
+  id: "0"
+  postID: "0"
+  title: comment
+Post:
+  body: This is the first post
+  createdAt: "14046"
+  creator: cosmos1g7x9cpj6w0jklshe3se57tlwydx6yfl8ex5g7n
+  id: "0"
+  title: Uno
+```
+
+## Delete comment
+
+```bash
+blogd tx blog delete-comment 0 0 --from alice -y
+```
+
+The results are output:
+
+```bash
+code: 0
+codespace: ""
+data: 0A270A252F636F736D6F6E6175742E626C6F672E626C6F672E4D736744656C657465436F6D6D656E74
+gas_used: "40125"
+gas_wanted: "200000"
+height: "253"
+info: ""
+logs:
+- events:
+  - attributes:
+    - key: action
+      value: DeleteComment
+    type: message
+  log: ""
+  msg_index: 0
+raw_log: '[{"events":[{"type":"message","attributes":[{"key":"action","value":"DeleteComment"}]}]}]'
+timestamp: ""
+tx: null
+txhash: 0312234CBB9EEA1A59D474496E100AFC5A460A0E60E7D009D3E9417530148A75
+```
+
+## Display
```

Comment:
@alijnmerchant21  can we give this heading a more descriptive name? What are we displaying?